packages/create-novu-app/templates/index.ts

```diff
@@ -166,6 +173,21 @@ export const installTemplate = async ({
     }
   }
 
+  /* write .env file */
+  const port = 4000;
+  const subdomain = buildBridgeSubdomain(apiKey);
+  const host = tunnelHost;
+
+  await fs.writeFile(
+    path.join(root, ".env"),
+    `PORT=${port}` +
+      os.EOL +
+      `TUNNEL_SUBDOMAIN=${subdomain}` +
+      os.EOL +
+      `TUNNEL_HOST=${host}` +
+      os.EOL,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Condider `Object.entries(obj).reduce(...)` with `obj` being a map of env var key to value.‏

Reply:
done